scripts/platform.sh

```diff
@@ -7,9 +7,6 @@ rm -rf platform lib/src/platform
 git clone https://github.com/opentdf/platform.git
 
 # Generate Typescript code
-cmd=$(npx buf --version)
-if [ $? -ne 0 ]; then
-  npm install @bufbuild/protoc-gen-es @bufbuild/buf @connectrpc/protoc-gen-connect-es
-fi
-npx buf generate platform/service
+cd lib
+npx buf generate ../platform/service
```

Comment:
Can we invoke this in `lib/package.json` from prepack, so we don't have to commit the generated ts files to the repo?

Reply:
So we do actually need to commit them to the repo because they are used by platform.ts internally for the types.  Otherwise platform.ts won't build